src/test/scala/io/iohk/ethereum/jsonrpc/EthJsonRpcControllerSpec.scala

```diff
@@ -0,0 +1,228 @@
+package io.iohk.ethereum.jsonrpc
+
+import io.iohk.ethereum.Fixtures
+import io.iohk.ethereum.db.components.{SharedEphemDataSources, Storages}
+import io.iohk.ethereum.domain.{Block, BlockchainImpl}
+import io.iohk.ethereum.jsonrpc.JsonSerializers.{BlockResponseSerializer, QuantitiesSerializer}
+import io.iohk.ethereum.network.p2p.messages.PV62.BlockBody
+import org.json4s.{DefaultFormats, Extraction}
+import org.json4s.JsonAST._
+import org.scalatest.{Matchers, WordSpec}
+import org.scalatest.prop.PropertyChecks
+import org.spongycastle.util.encoders.Hex
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+class EthJsonRpcControllerSpec extends WordSpec with Matchers with PropertyChecks {
```

Comment:
It agree it sounds better to do it that way, so as to separate the testing. Also, my previous version of the tests were done as you recommend for `EthServiceSpec` so I'll be able to have `JsonRpcControllerSpec` and `EthServiceSpec` tests quite fast :+1: